markdown/blog/gravitee-sponsorship-2025.md

```diff
@@ -0,0 +1,63 @@
+Gravitee has long been part of AsyncAPI’s journey, not just as a sponsor but as an adopter, contributor, and champion of event-driven architectures (EDAs) that are secure, discoverable, and truly developer-first.
+
+## A Shared Vision: Open Collaboration, Stronger Infrastructure
+
+AsyncAPI’s mission is to make working with event-driven systems as seamless as it is working with REST. That mission only becomes a reality when organizations across industries join forces to push it forward sharing knowledge, building tooling, contributing to the spec, and supporting our diverse global community.
+Gravitee’s sponsorship is rooted in this shared vision. They understand that success in the API economy depends not only on robust tooling but on shared specifications and interoperable ecosystems. That’s why Gravitee has made AsyncAPI an essential part of its architecture, empowering teams to define, document, and govern their asynchronous APIs just as easily as their REST endpoints.
+
+## More Than Sponsorship: Investing in Community-Driven Growth
+
+Gravitee’s renewed platinum sponsorship isn’t just a logo on a banner, it’s a commitment to sustainable, community-first growth. In 2024, their support helped us organize inclusive, impactful gatherings like AsyncAPI Conf Online and the Conference on Tour series now branded as the AsyncAPI Conference.
+In 2025, their contribution helps us go even bigger, with in-person conferences in Lagos, Munich, Bangalore, London, Paris, and Online each designed to bring the AsyncAPI community closer together across regions and time zones. Gravitee’s support enables:
+
+- Operational and travel cost for events.
+- Broader outreach and visibility in growing the AsyncAPI communities around the world.
+- Resources for speakers and panelists from diverse backgrounds.
+
+When we say Gravitee is helping us “hold nothing back,” we mean it, they’re helping build an ecosystem where everyone can contribute, learn, and grow, no matter their location or technical background.
+
+## Gravitee + AsyncAPI: A Real-World Use Case
+
+Behind Gravitee’s sponsorship is a real, practical investment in using the AsyncAPI specification to build and manage asynchronous APIs. Gravitee enables organizations to work seamlessly across protocols like [Kafka](https://kafka.apache.org/), [MQTT](https://mqtt.org/), and [WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) using the AsyncAPI spec as a foundation for documentation, governance, and developer experience. This approach strengthens discoverability and observability across distributed systems, making Gravitee a natural ally in advancing AsyncAPI’s mission and ecosystem.
```

Comment:
There's repetition with line 28. Make section/line 28 focus on sponsorship, and then section/line 38 focus on use cases.